src/main/resources/assets/opencomputers/loot/mineos/installer.lua

```diff
@@ -0,0 +1,385 @@
+local component = require("component")
+local computer = require("computer")
+local term = require("term")
+local unicode = require("unicode")
+local event = require("event")
+local fs = require("filesystem")
+local seri = require("serialization")
+local shell = require("shell")
+local gpu = component.gpu
+
+-----------------Проверка компа на соответствие сис. требованиям--------------------------
+
+shell.execute("cd ..")
+shell.setWorkingDirectory("")
+
+--Создаем массив говна
+local govno = {}
```

Comment:
Can't you just name that variable "errors" or "problems"? And i know what that word means, thank you very much for a short explanation of a single russian word.